virttest/env_process.py

```diff
@@ -558,6 +558,47 @@ def _call_image_func():
         _call_image_func()
 
 
+def check_host_cpnt_version(required_cpnt, cpnt, get_version_func, test):
+    """"
+    Check if a component on host satisfied the test component requirements.
+    If not, cancel the test.
+
+    :param required_cpnt: A tuple like string of required component.
+                          (None,"2.10.0") means the cpnt version must
+                          less than or equal to 2.10.0".
+                          ("2.1.0",None) means the cpnt version must
+                          greater than 2.1.0".
+                          ("2.1.0","2.10.0") means the cpnt version must
+                          less than or equal to 2.10.0" and greater than
+                          2.1.0".
+    :param cpnt: A raw component version string.
+    :param get_version_func: A function for getting actual version number.
+    :param test: A test object.
+    """
+    def _compare_version(version, verison_range):
+        from distutils.version import LooseVersion
```

Comment:
How about implementing our own one since the pylint issue is less likely to be solved soon.

Reply:
Please don't try to reinvent the wheel. This is known issue https://github.com/PyCQA/pylint/issues/73 and you can simply add `# pylint: disable=no-name-in-module,import-error`